blog-posts/case-fc-bayern.md

```diff
@@ -0,0 +1,66 @@
+---
+path: "/blog/case-fc-bayern/"
+date: "2022-02-21"
+title: "Relaunch FC Bayern Website"
+featuredImage: images/fcb/hero.png
+author: Fabian Dietenberger
+authorSummary: "Frontend Engineer, Satellytes"
+seoMetaText: Our Approach to relaunch the Frontend of the Fc Bayern Website.
+teaserText: Our Approach to relaunch the Frontend of the Fc Bayern Website.
```

Comment:
@esreverse Those are the text snippets for the teaser and for google. Here a screenshot from the teaser. What do you want to read?

![Screenshot 2022-02-22 at 06 07 44](https://user-images.githubusercontent.com/1701755/155066811-04f8a601-cfba-403b-b3b2-ced7dc148910.png)